app/views/case-progress.html

```diff
@@ -14,21 +14,25 @@
 
 <table class="non-javascript">
     <tr>
+        <th>Name</th>
         <th>Community Last 16 Weeks</th>
         <th>License Last 16 Weeks</th>
         <th>Total Cases</th>
         <th>Warrants Total</th>
         <th>Unpaid Work Total</th>
         <th>Overdue Terminations Total</th>
     </tr>
+    {% for caseProgress in caseProgressList%}
     <tr>
+        <td>{{ caseProgress.name }}</td>
```

Comment:
This should be a `<th>` because it's a header I think. Perhaps @stevenadams could shed some light on this?